test/plugs_test.py

```diff
@@ -37,6 +38,7 @@ def _asdict(self):
 
   def increment(self):
     self.number += 1
+    self.notify_update()
```

Comment:
can you check that if they attempt to do `notify_update` without the `super(...).__init__()` then we raise a more specific error?